### ISSUE TITLE
lost alignment between columns

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#78]
Originally reported by: **ignasi (Bitbucket: [ignasi](https://bitbucket.org/ignasi), GitHub: [ignasi](https://github.com/ignasi))**

----------------------------------------

Hi,

I've found a little problem with top aligned columns in a frame.
Please, test attached example. You will see that itemized environments are not aligned when the frame has [t] option and
they look good with the centered frame default option.
 
Do you know what's wrong?
 
Thank you,
 
Ignasi

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/78

**Comment by josephwright** [comment]
*Original comment by* **Vedran Miletić (Bitbucket: [rivanvx](https://bitbucket.org/rivanvx), GitHub: Unknown)**:

----------------------------------------

I know how to fix it, and it's quite easy, but I'm not sure if it will break something else. If you need the fix badly, replace lines 256--260 in beamerbaseframe.sty which read:
    
      \def\beamer@initfirstlineunskip{%
        \def\beamer@firstlineitemizeunskip{%
          \vskip-\partopsep\vskip-\topsep\vskip-\parskip%
          \global\let\beamer@firstlineitemizeunskip=\relax}%
        \everypar{\global\let\beamer@firstlineitemizeunskip=\relax}}
    
with
    
      \def\beamer@initfirstlineunskip{}%
    
This will fix it.

**Comment by josephwright** [comment]
*Original comment by* **svenkoehler (Bitbucket: [svenkoehler](https://bitbucket.org/svenkoehler), GitHub: [svenkoehler](https://github.com/svenkoehler))**:

----------------------------------------

Why is this still on hold? I just hit this bug and an itemize in columns on a top-align frame really looks very bad!

A workaround that can be copy/pasted into the document itself is the following:

```
#!latex
\makeatletter
%%% taken from beamerbaseframe.sty
\define@key{beamerframe}{t}[true]{% top
  \beamer@frametopskip=.2cm plus .5\paperheight\relax%
  \beamer@framebottomskip=0pt plus 1fill\relax%
  \beamer@frametopskipautobreak=\beamer@frametopskip\relax%
  \beamer@framebottomskipautobreak=\beamer@framebottomskip\relax%
  \def\beamer@initfirstlineunskip{}%
}
\makeatother
```

which was suggested here:
http://tex.stackexchange.com/questions/16447/beamer-top-aligning-columns-within-a-top-aligned-frame

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

I'll take a look at this

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

Drop special treatment of first itemise spacing (fixes #78)

I'm leaving the code which allows the rest of the mechanism to work:
we don't use it with this change, but it's not entirely clear what
is really going on (there are no notes anywhere in the code).
